cel/options.go

```diff
@@ -376,3 +378,83 @@ func EvalOptions(opts ...EvalOption) ProgramOption {
 		return p, nil
 	}
 }
+
+func fieldToCELType(field protoreflect.FieldDescriptor) (*exprpb.Type, error) {
+	// https://github.com/google/cel-spec/blob/master/doc/langdef.md#protocol-buffer-data-conversion
+	switch (field).Kind() {
```

Comment:
Consider using `common/types/pb/checked.go` as the source of truth for type conversions and just group this into two chunks:

```go
if field.Kind() == protoreflect.MessageKind {
  msgName := field.Message().FullName()
  if type, found := pb.CheckedWellKnowns[msgName]; found {
    return type, nil
  }
  return decls.NewObjectType(msgName), nil
}
if type, found := pb.CheckedPrimitives(field.Kind()); found {
  return type, nil
}
return nil, <err>
```

Reply:
Thanks, that worked (with an extra case for enum)